pyproject.toml

```diff
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2024 Margo
+#
+# SPDX-License-Identifier: MIT
+#
+# Author: Michael Adler <michael.adler@siemens.com>
```

Comment:
Nitpicking, but that should read
`// SPDX-FileContributor: Contributor Name <contributor.email@company.com>`
as per the [(Public) Code Repositories Guideline](https://github.com/margo#public-code-repositories)